packages/exhibitor/extra/start_exhibitor.py

```diff
@@ -49,6 +54,7 @@ def invoke_detect_ip():
     '-Djava.util.prefs.userRoot=/var/lib/dcos/exhibitor/',
     '-Duser.home=/var/lib/dcos/exhibitor/',
     '-Duser.dir=/var/lib/dcos/exhibitor/',
+    '-Djna.tmpdir=%s' % jna_tmpdir,
```

Comment:
Sadly systemd doesn't support this. the `Environment=` directive does not perform environment variable substitution.